downloads.md

```diff
@@ -200,6 +200,20 @@ You can add the following dependencies to your `pom.xml` to include Apache Flink
 The `statefun-sdk` dependency is the only one you will need to start developing applications.
 The `statefun-flink-harness` dependency includes a local execution environment that allows you to locally test your application in an IDE.
 
+### Apache Flink ML
+
+You can add the following dependencies to your `pom.xml` to include Apache Flink ML in your project.
+
+```xml
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-ml-uber_2.12</artifactId>
+  <version>{{ site.FLINK_ML_VERSION_STABLE }}</version>
+</dependency>
+```
+
+The `flink-ml-uber_2.12` is a whole set of dependencies required for both using and developing algorithms. To only use existing algorithms, users could also only depend on `flink-ml-lib_2.12` and to only develop new algorithms, user could also only depend on `flink-ml-core` and `flink-ml-iteration` (if the algorithm is iterable).
```

Comment:
This paragraph seems a bit difficult to read.

Could we use the following words:

`flink-ml-uber_2.12` provides all the dependencies needed to use Flink ML. Alternatively, advanced users could use the following artifacts to import a minimal set of Flink ML dependencies for their target use-cases:

- Use artifact `flink-ml-core` in order to develop custom ML algorithms.
- Use artifacts `flink-ml-core` and `flink-ml-iteration` in order to develop custom ML algorithms which require iteration.
- Use artifact `flink-ml-lib_2.12` in order to use the off-the-shelf ML algorithms from Flink ML.